src/mongoose_instrument_probe_cets.erl

```diff
@@ -0,0 +1,85 @@
+-module(mongoose_instrument_probe_cets).
+-behaviour(mongoose_instrument_probe).
+-include("mongoose_logger.hrl").
+
+-export([start/0, stop/0]).
+
+%% Probe callbacks
+-export([probe/2, instrumentation/0]).
+-ignore_xref([stop/0, instrumentation/0]).
+
+start() ->
+    mongoose_instrument:set_up(instrumentation()).
+
+stop() ->
+    mongoose_instrument:tear_down(instrumentation()).
+
+all_zeros() ->
```

Comment:
I don't see a reason to put all the zeros here. An empty measurement is still a valid one. And it's closer to the truth, because we don't actually know the values.